lib/OpenQA/Schema/Result/Jobs.pm

```diff
@@ -948,6 +958,11 @@ sub update_status {
 
     if ($self->worker) {
         $self->worker->set_property("INTERACTIVE", $status->{status}->{interactive} // 0);
+        # mark the worker as alive
+        $self->worker->seen;
```

Comment:
The problem is we call seen on undefined and I don't get how this is even possible - so I moved the seen call in an if. That's all